gen_tests.sh

```diff
@@ -2,21 +2,21 @@
 
 set -e
 
-mvn clean package -pl wire-compiler -am -Dmaven.test.skip
+./gradlew -p wire-compiler clean installDist
```

Comment:
This whole script can die and be replaced with `JavaExec` tasks and actual compilation of the generated sources once this actually lands.